client/auth/emailed-login-link-expired.jsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import EmptyContent from 'components/empty-content';
+import RedirectWhenLoggedIn from 'components/redirect-when-logged-in';
+
+class EmailedLoginLinkExpired extends React.Component {
+	render() {
+		const { translate } = this.props;
+
+		return (
+			<div>
+				<RedirectWhenLoggedIn
+					redirectTo="/"
+					replaceCurrentLocation={ true }
+				/>
+				<EmptyContent
+					action={ translate( 'Return to WordPress.com' ) }
+					actionURL={ '/' }
+					illustration={ '/calypso/images/drake/drake-404.svg' }
+					illustrationWidth={ 500 }
+					line={ translate( 'Maybe try resetting your password instead' ) }
+					secondaryAction={ translate( 'Reset my password' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **13** times:
`translate( 'Reset My Password' )` *ES Score: 13*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/try/logged-out-magic-li-req-page)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).